Sprint-1/refactor/includes.js

```diff
@@ -1,13 +1,15 @@
 // Refactor the implementation of includes to use a for...of loop
 
 function includes(list, target) {
-  for (let index = 0; index < list.length; index++) {
-    const element = list[index];
-    if (element === target) {
-      return true;
-    }
-  }
-  return false;
+    // if (list.includes(target)) {
+    //   return true;
+    // }
+    // else{
+    //   return false;
+    // }
+    return list.includes(target)
```

Comment:
While this solution works, the spec does have the following requirement:
`// Refactor the implementation of includes to use a for...of loop`